my-app/components/Section-one/_Section-one.tsx

```diff
@@ -1,26 +1,31 @@
 import Image from 'next/image'
+import { Title } from './_Section-one-style';
+import Card from '../Card/_Card'
 
-const Menu = () =>{
+const SectionOne = () =>{
     return (
-        <header className='row'>
-            <div className='col-xs-2'>
-                <Image 
-                    src='/logo.png'
-                    alt='logo'
-                    width={50}
-                    height={50}
-                />
+        <header>
+            <div className='row center-xs'>
+                <Title className='col-xs-12'>
+                    <div></div>
```

Comment:
Essa div seria um separador? Se sim, precisa criar um styled components pra isso. Se não for um separador, ela não serve